lib/src/common/observer_controller.dart

```diff
@@ -277,21 +277,23 @@ mixin ObserverControllerForScroll on ObserverControllerForInfo {
   /// Jump to the specified index position without animation.
   ///
   /// If the height of the child widget and the height of the separator are
-  /// fixed, please pass the [isFixedHeight] parameter.
+  /// fixed, please pass the [isFixedHeight] parameter and the
+  /// [renderSliverType] parameter.
   ///
   /// If you do not pass the [isFixedHeight] parameter, the package will
   /// automatically gradually scroll around the target location before
   /// locating, which will produce an animation.
```

Comment:
Could you please add a description of the renderSliverType parameter?

Reply:
copy that